### ISSUE TITLE
Issues when enableing

### ISSUE THREAD

**Issue by Bimi124** [Zylann/godot_heightmap_plugin#266]
**Describe the bug**
I can't enable the plugin for the first time (but then it works)

**To Reproduce**
1. Downoad v1.5.2 from the asset store
2. Put the `plugins` folder in your project folder
3. Wait for godot to import the addon
4. Go to `project > project settings > plugins`
5. Try to enable `Heightmap Terrain`, version `1.5.2` by `Marc Gilleron`
6. Go to GitHub and create an Issue (I don't thing you *need* to do that)
7. Enable it! Finally!
Side note: I used GLES2 but it worked the second time I tried

**Expected behavior**
The plugin becomes enabled with no error messages first time

**Screenshots**
(I tried to take some but it works now...)

**Environment**
- OS: Windows
- Graphics card: see log below
- Godot version: see log below
- Plugin version: 1.5.2
- Renderer used: GLES2 but I switched to GLES3 after sucessfully enableing it

**Error log**
Error log before enableing it: https://pastebin.com/U2VGKd0g

**Comment by Zylann** [comment]
> I used GLES2 but it worked the second time I tried

The plugin does not work with GLES2

About the import errors, see https://github.com/godotengine/godot/issues/23937

**Comment by Bimi124** [comment]
So yo're telling me your plugin can't *load in* because I'm using GLES2 but the second time I try to load it in using GLES2 it does work? 😕

**Comment by clemens-tolboom** [comment]
Reading the log (why should I visit another site for that? :-/)

> Godot Engine v3.2.3.stable.official - https://godotengine.org
> OpenGL ES 3.0 Renderer: Intel(R) HD Graphics 4000

```
ERROR: Cannot open file 'res://.import/grass_quad.obj-da067750350fe98ec466261b2aeaf486.mesh'.
   At: core/io/resource_format_binary.cpp:1047
ERROR: Cannot open file 'res://.import/grass_quad_x2.obj-2054c140f543f2a80e2eb921f865ea49.mesh'.
   At: core/io/resource_format_binary.cpp:1047
ERROR: Cannot open file 'res://.import/grass_quad_x3.obj-8691724bc5006b6f65d4e8742ffc84dc.mesh'.
   At: core/io/resource_format_binary.cpp:1047
ERROR: Cannot open file 'res://.import/grass_quad_x4.obj-c449a7d6c810ba1595ed30df9fbf3d28.mesh'.
   At: core/io/resource_format_binary.cpp:1047
```

and (multiple times on shader code)

> SCRIPT ERROR: (null): "sampler2DArray" type is supported only on high-end platform!

I guess the `ERROR` links to Godot like @Zylann said. (I wasn't aware of switching repo)

But as this relate to Godot Engine and is still open can we close it: old, outdated Godot version?

I started reading this hoping to moving forward https://github.com/godotengine/godot/issues/23937

**Comment by Bimi124** [comment]
I wouldn't mind if it was closed. I'll just do it because I'm writing a comment now.
I posted this issue two years ago and it may very well be fixed by now. If not, someone could just reopen it or create a new one.
Since you've asked about why i uploaded the logs to Pastebin: Because it's an easy tool to use. The log is extremely long and adding it to the issues description would've made it so much longer. I don't know why I haven't used GitHub's built-in file uploading tool. I might haven't had so much experience with the website given the fact that I had created this account very recently back then.
